examples/clip/clip/clip.py

```diff
@@ -0,0 +1,172 @@
+import hashlib
+import os
+import urllib
+import warnings
+from typing import Any, List, Union
+
+from PIL import Image
+from pkg_resources import packaging
+from tqdm import tqdm
+
+import mindspore as ms
+from mindspore import Tensor, load_checkpoint
+from mindspore.dataset.transforms import Compose
+from mindspore.dataset.vision import CenterCrop, Normalize, Resize, ToPIL, ToTensor
+
+from .model import build_model
+from .simple_tokenizer import SimpleTokenizer as _Tokenizer
+
+try:
+    from mindspore.dataset.vision import Inter
+
+    BICUBIC = Inter.BICUBIC
+except ImportError:
+    BICUBIC = Image.BICUBIC
+
+if packaging.version.parse(ms.__version__) < packaging.version.parse("2.0.0"):
+    warnings.warn("MindSpore version 2.0.0 or higher is recommended")
+
+__all__ = ["available_models", "load", "tokenize"]
+_tokenizer = _Tokenizer()
+
+_MODELS = {
+    "RN50": "https://openaipublic.azureedge.net/clip/models/afeb0e10f9e5a86da6080e35cf09123aca3b358a0c3e3b6c78a7b63bc04b6762/RN50.pt",
```

Comment:
We should use mindspore *.ckpt by default, which can be updated after uploading the converted weights to download servers.